### PR TITLE
Include a section on chart colours

### PR DESCRIPTION
**Pull request by ella-goodman**
I have added in this change, as the problem with using the GOV.UK design system colours which are already on this page for charts is that, depending on what order the colours are presented in, they would not meet the adjacent colour contrast requirements.

The colours guidance I have linked was developed by a cross government working group led by the Department for Education. The colour palette in this guidance have been designed for use in standard line, bar and pie charts. When all the guidance is followed, the colours mentioned in the Analysis Function colour palette will create charts which are as accessible as possible. This  colour palette is used in the Whitehall publisher on GOV.UK.

### CODE REVIEW

**Comment by ella-goodman** [comment]
Hi I have been in discussion with Steve from GDS Support services and he asked me to post on here to add some more detail as to why we have asked for this change.  

I work in the Analysis Function (AF) Central Team, we assist people across government with data presentation to help make the content they publish align with the Web Content Accessibility Guidelines (WCAG).  

We deal with data presentation rather than website design. However, we received some emails of confusion over how the [GOV.UK design system colours page](https://design-system.service.gov.uk/styles/colour/) relates to creating charts. It seems this page is getting confused as a page for recommended colours for anything to go on the GOV.UK website including charts and data visualisations. We want to make sure the message is clear and consistent across government by linking to the [AF colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

The AF colours guidance was developed by a cross government working group led by the Department for Education. It includes a thoroughly tested [colour palette which is used in the Whitehall publisher](https://www.gov.uk/government/publications/examples-of-visual-content-to-use-on-govuk/examples-of-visual-content-to-use-on-govuk). The AF colours guidance is also already linked to from the [Content design: planning, writing and managing content](https://www.gov.uk/guidance/content-design/images) on GOV.UK. We want to make sure all the help across government is linked up and in line with WCAG.

The problem with using the GOV.UK design system colours which are already on this page for charts is that, depending on what order the colours are presented in, they would not meet the adjacent colour contrast requirements.
[Success Criterion 1.4.11](https://www.w3.org/TR/WCAG21/#non-text-contrast) Non-text Contrast states: The visual presentation of user interface components and graphical objects have a contrast ratio of at least 3:1 against adjacent colours.
Adjacent colours include the graphical object colour against a white background. I have therefore, tested the colours in the colour palette mentioned on the [GOV.UK Styles Colour page](https://design-system.service.gov.uk/styles/colour/) using the [WebAIMS contrast checker](https://webaim.org/resources/contrastchecker/) against white. This showed the yellow, mid grey, light pink and orange do not meet this 3:1 contrast requirement. These colours would not therefore be suitable for graphical objects and user interface components if the background is white.

I hope this adds some context to my request. Any further queries please contact the Analysis.Function@ons.gov.uk inbox. 
Best wishes, Ella

**Comment by christopherthomasdesign** [comment]
Thanks for proposing this @ella-goodman !

I think given that this is the standard now used on GOV.UK and more widely, it makes sense for us to back it and make it clear.

So I'm generally happy to approve the change, could you do a more detailed content review @calvin-lau-sig7 to make sure it's written consistently with the rest of our guidance?

File: src/styles/colour/index.md [hunk 1]
```diff
@@ -94,3 +94,6 @@ You can find departmental colours in the GOV.UK Frontend [_colours-organisations
   {% endfor %}
  </tbody>
 </table>
+
+### Colour palette for charts
+When creating charts refer to the colour palettes and guidance set out in the Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).
```
Comment (calvin-lau-sig7) [inline src/styles/colour/index.md:99]:
```suggestion
When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

The colour palettes recommended by the Government Analysis Function are based the colours shown on this page, but they've made some slight changes to improve colour contrast.
```

**Review by calvin-lau-sig7 — COMMENTED** [review]
Hi @ella-goodman! Could we publish this change with this adapted line from Analysis Function's guidance? I think it explains the relationship between the colour palettes really well.

**Comment by ella-goodman** [comment]
Hi @calvin-lau-sig7 thanks for looking at this, would you mind perhaps adapting line 101 to say : 
"The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They have made some slight changes to improve colour contrast. When all the guidance is followed, the colours mentioned in the Analysis Function colour palette will create charts which meet the Web Content Accessibility Guidelines (WCAG)."

Just to add in some context as to why the colour contrast needed to be changed. Best wishes, Ella

File: src/styles/colour/index.md (hunk 1 above)
Comment (calvin-lau-sig7) [reply src/styles/colour/index.md:99]: > 

```suggestion
When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).

The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast. When all the guidance is followed, the colours mentioned in the Analysis Function colour palette will create charts which meet the Web Content Accessibility Guidelines (WCAG).
```

**Review by calvin-lau-sig7 — APPROVED** [review]

**Comment by calvin-lau-sig7** [comment]
Looks to me, thanks @ella-goodman! I'm usually a bit cautious about making an open statement that something will meet WCAG, but what you've written feels pretty watertight.

Going to see if @christopherthomasdesign can give an approve from the team as well.

File: src/styles/colour/index.md [hunk 2]
```diff
@@ -94,3 +94,8 @@ You can find departmental colours in the GOV.UK Frontend [_colours-organisations
   {% endfor %}
  </tbody>
 </table>
+
+### Colour palette for charts
+When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).
+
+The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast. When all the guidance is followed, the colours mentioned in the Analysis Function colour palette will create charts which meet the Web Content Accessibility Guidelines (WCAG).
```
Comment (calvin-lau-sig7) [inline src/styles/colour/index.md:101]:
```suggestion
The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast, in line with the requirements of the Web Content Accessibility Guidelines (WCAG).
```

**Comment by calvin-lau-sig7** [comment]
Hi @ella-goodman, that line looks good to me! I've committed the changes into this pull request. 
With your okay or thumbs up I'll work to get this merged and published.

**Comment by ella-goodman** [comment]
That looks good, thanks Calvin. Best wishes, Ella

File: src/styles/colour/index.md [hunk 3]
```diff
@@ -94,3 +94,8 @@ You can find departmental colours in the GOV.UK Frontend [_colours-organisations
   {% endfor %}
  </tbody>
 </table>
+
+### Colour palette for charts
+When creating charts, use the colour palettes and guidance set out in the Government Analysis Function [Data visualisation: colours guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/).
+
+The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast, in line with the requirements of the Web Content Accessibility Guidelines (WCAG).
```
Comment (calvin-lau-sig7) [inline src/styles/colour/index.md:101]:
```suggestion
The colour palettes recommended by the Government Analysis Function are based on the colours shown on this page. They've made some slight changes to improve colour contrast, in line with the Web Content Accessibility Guidelines (WCAG).
```

**Comment by calvin-lau-sig7** [comment]
Thanks @ella-goodman! I'm going to merge shortly with a minor edit to simplify the WCAG mention, as the team's fed back that "requirements" wasn't quite an accurate way to describe WCAG without further explanation about legislation, etc. Hope that looks good.

**Comment by ella-goodman** [comment]
Thanks @calvin-lau-sig7

**Pull request merged**